### ISSUE TITLE
what's fine-tune mean actually

### ISSUE THREAD

**Issue by tsungruihon** [mozilla/TTS#198]
Please consider posting on [TTS Discourse](https://discourse.mozilla.org/c/tts) page, if your issue is not directly related to TTS development process.

Happy posting!

https://discourse.mozilla.org/c/tts

I have noticed that in `readme`, you said 

> Constant history window. Instead of using only the last frame of predictions, define a constant history queue. It enables training with gradually decreasing prediction frame (r=5 --> r=1) by only changing the last layer. For instance, you can train the model with r=5 and then fine-tune it with r=1 without any performance loss. It also solves well-known PreNet problem #50.

May i ask what's the meaning of `fine-tune` , and how could i actual implement this approach ?

**Comment by erogol** [comment]
This should go to discourse page, as you appended above :)

Fine-tunning is to retrain a pre-trained model possibly with a new dataset.
